hack/verify-vendor.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+WMCO_ROOT=$(pwd)
+
+# create a temporary location to put our new vendor tree
+mkdir -p "${WMCO_ROOT}/_tmp"
+_tmpdir="$(mktemp -d "${WMCO_ROOT}/_tmp/wmco-vendor.XXXXXX")"
```

Comment:
Why is `XXXXX` needed?

Reply:
We need it when templates are not given in [mktemp](https://www.gnu.org/software/autogen/mktemp.html)